src/services/SQSService.js

```diff
@@ -39,12 +49,24 @@ class SQSService {
       const template = await NotificationTemplateModel
         .findOne({ _id: mongoose.Types.ObjectId(type.template_id) })
         .exec();
+
+      const generatedTemplate = Mustache.render(template, { message: JSON.parse(event.Records[0].body).text });
```

Comment:
I don't know how this would work if the template variable holds a MongoDB document

Reply:
done